tests/unit/components/Asset.spec.js

```diff
@@ -185,7 +185,7 @@ describe('Asset', () => {
       expect(asset.exists()).toBe(true);
       expect(asset.props()).toEqual({
         alt: image.alt,
-        loading: "lazy",
```

Comment:
Lets revert these as they are not related to the PR.

Reply:
Yea I figured. These 2 are generated when I ran test.

Reply:
Should we open a separate PR to fix these?